paasta_tools/cli/cmds/logs.py

```diff
@@ -1388,8 +1391,8 @@ def paasta_logs(args: argparse.Namespace) -> int:
             service=service,
             levels=levels,
             components=components,
-            clusters=clusters,
-            instances=instances,
+            clusters=cluster,
+            instances=instance,
```

Comment:
It feels a little weird to me that this still works.  For clusters, I think the issue is that you are setting `nargs=1` for the `--cluster` argument, which causes it to return a 1-element list instead of a string.  I think that is roughly ok, but it'd be better if we just told it that `--cluster` was a `str` and then passed `clusters=[args.cluster]` or whatever here.  This isn't a huge deal, but would improve clarity a bit.

For instances, I think there are actually two weird things going on here that we should fix.  `instance` is now a `str` whereas previously we were passing a list of strings.  However, all the logs functions specify that `instances` is an `Iterable[str]`.  What they meant was something like `['instance1', 'instance2']`, but because strings are actually made up of shorter strings, it turns out that just a regular old string like `'instance1'` is actually also an `Iterable[str]`.  So this is passing its type check, but for counter intuitive reasons.

Even weirder, I was wondering how this was passing the `filter_fn` which checks that `parsed_line.get("instance") in instances` (see `paasta_app_output_passes_filter`).  Before, `instances` would have been a list of strings.  Now it is just a string.  But it all ends up working because the `in` is now checking whether the instance from the log line is a substring of the instance provided as an argument.  Very weird.

Anyway, I think we should do the same thing for instances -- pass `instances=[instance]` to these functions so that they behave the way we expect them to.